presto-docs/src/main/sphinx/develop/functions.rst

```diff
@@ -386,3 +386,124 @@ scalar and aggregation functions.
             @BlockPosition @SqlType("T") Block block,
             @BlockIndex int index) { /* ...implementation */ }
 
+Complex Type Function Descriptor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Complex type function descriptor is used for specifying the metadata for the map or array functions that deal with arguments of ``ROW`` types. Presto optimizer tries to collect
+information about accessed subfields as granular as possible so tha file format reader (ORC/DWRF/Parquet) could prune all subfields that are irrelevant for execution of the current
+query. Consider the example below.
+
+.. code-block:: sql
```

Comment:
Please fix the code block, and if you can, build the docs locally and review them for correct formatting. 
(See "Building manually" and "Viewing the documentation" in the [presto-docs README](https://github.com/prestodb/presto/tree/master/presto-docs) for how to build and view the docs locally. )

A local build of this doc looks like this: 
<img width="958" alt="Screenshot 2024-01-02 at 2 03 17 PM" src="https://github.com/prestodb/presto/assets/7013443/f23e9593-5b38-4366-b14d-4c135fcfed09">

Reply:
fixed